### PR TITLE
Enable extrakeys, mousekeys for all VIA keymaps.

### PR DESCRIPTION
**Pull request by kaylanm**
## Description

Thanks to @yiancar's V-USB work, `EXTRAKEY_ENABLE` and `MOUSEKEY_ENABLE` should be safe to no longer turn off on VIA keymaps running on atmega32a platforms. It looks like these flags also leaked into some atmega32u4 boards, so I've removed them there as well. In a few cases the firmware was too large, and `CONSOLE_ENABLE`/`COMMAND_ENABLE` was turned off to compensate. This has been tested on a singa R2 board - please let me know if it's too overbroad of a change.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* None

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/lazydesigners/dimple/keymaps/via/rules.mk [hunk 1]
```diff
@@ -7,4 +7,3 @@ MIDI_ENABLE = no
 BLUETOOTH_ENABLE = no
 KEY_LOCK_ENABLE = no
 TERMINAL_ENABLE = no
```
Comment (fauxpark) [inline keyboards/lazydesigners/dimple/keymaps/via/rules.mk:9]:
```suggestion
```

These + MIDI, Faux Clicky and Leader are disabled by default.

File: keyboards/thevankeyboards/minivan/keymaps/via/rules.mk [hunk 2]
```diff
@@ -7,4 +7,3 @@ MIDI_ENABLE = no
 BLUETOOTH_ENABLE = no
 KEY_LOCK_ENABLE = no
 TERMINAL_ENABLE = no
```
Comment (fauxpark) [inline keyboards/thevankeyboards/minivan/keymaps/via/rules.mk:9]:
```suggestion
```

As above.

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/lazydesigners/dimple/keymaps/via/rules.mk (hunk 1 above)
Comment (kaylanm) [reply keyboards/lazydesigners/dimple/keymaps/via/rules.mk]: Removed

File: keyboards/thevankeyboards/minivan/keymaps/via/rules.mk (hunk 2 above)
Comment (kaylanm) [reply keyboards/thevankeyboards/minivan/keymaps/via/rules.mk]: Removed

**Review by fauxpark — APPROVED** [review]

**Review by zvecr — APPROVED** [review]

**Comment by zvecr** [comment]
Thanks!

**Pull request merged**